packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-scorecard.md

```diff
@@ -4,12 +4,16 @@ section: accessibility
 ---
 import { AccessibilityScorecard } from './accessibility-scorecard.js';
 
+## Scoring your accessibility
+
+To support the proper assessment of accessibility measures, we've created a scorecard that outlines the essential criteria to meet. You can use this scorecard as a guide to evaluate the state of accessibility for any UI built using PatternFly.
```

Comment:
I'm wondering if we should say something about how these are the way in which we test, not just criteria per say? Maybe something like, "we've created a scorecard that outlines the ways that we recommend testing for accessibility and how your UI meets those expectations." That probably needs word-smithing. 🙃